monitoring/uss_qualifier/scenarios/astm/utm/dss/authentication/authentication_validation.py

```diff
@@ -173,6 +195,44 @@ def _ensure_clean_workspace_step(self):
         self.end_test_step()
 
     def _ensure_test_entities_dont_exist(self):
+
+        # Drop OIR's first: subscriptions may be tied to them and can't be deleted
+        # as long as they exist
+        # TODO cleanly move this into the test fragments once the relevant PRs (notably #535) are merged
+        with self.check(
+            "Operational intent references can be queried by ID", self._pid
+        ) as check:
+            try:
+                oir, q = self._dss.get_op_intent_reference(self._test_id)
+                self.record_query(q)
+            except QueryError as qe:
+                self.record_queries(qe.queries)
+                if qe.queries[0].response.status_code == 404:
+                    return  # All is good
+                else:
+                    query = qe.queries[0]
+                    check.record_failed(
+                        summary=f"Could not query OIR {self._test_id}",
+                        details=f"When attempting to query OIR {self._test_id} from the DSS, received {query.response.status_code}: {qe.msg}",
```

Comment:
nit, this should be enough:
```suggestion
                        details=f"When attempting to query OIR {self._test_id} from the DSS, received {query.response.status_code}: {qe}",
```